code/modules/roguetown/roguejobs/engineer/prostheticarm.dm

```diff
@@ -79,14 +79,14 @@
 /obj/item/bodypart/r_arm/prosthetic/bronzeright
 	name = "Bronze right arm"
 	desc = "A replacement right arm, engineered out of bronze."
-	icon = 'icons/roguetown/items/misc.dmi' //CHANGE THIS
-	icon_state = "prarm" //THIS TOO
+	icon = 'icons/roguetown/items/misc.dmi'
+	icon_state = "prarm" //UPDATE THIS WITH SPRITE SOON
 	resistance_flags = FIRE_PROOF
 	obj_flags = CAN_BE_HIT
 	status = BODYPART_ROBOTIC
 	brute_reduction = 0
 	burn_reduction = 0
-	max_damage = 20
+	max_damage = 100
```

Comment:
and by a touch I mean...lets say
```suggestion
	max_damage = 110
```
It's metal. Very intricate metal. But still metal. Just enough warrant gamers with to much coin and time becoming quad metal limb warriors.

Reply:
I considered it, but bronze ones are only replacements for now. However, blacksteel prosthetics will exist in the future, which will be more durable than this. 

There will also be silver/gold ones for those particularly luxurious nobles.

Though, you do make a good point with just chopping off a grag's arms instead of having one made, so Ill agree to this for now.